index.html

```diff
@@ -4364,9 +4394,10 @@ <h2>
         <p>
           The <dfn>update a <code>PaymentRequest</code>'s details
           algorithm</dfn> takes a <a>PaymentDetailsUpdate</a>
-          <var>detailsPromise</var> and a <a>PaymentRequest</a>
-          <var>request</var>. The steps are conditional on the
-          <var>detailsPromise</var> settling. If <var>detailsPromise</var>
+          <var>detailsPromise</var>, a <a>PaymentRequest</a>
+          <var>request</var>, and optionally a DOMString <var>identifer</var>
+          (a <a>payment method identifier</a>). The steps are conditional on
```

Comment:
I don't think it should be optional. It should just either be a string or null. Spec-ese doesn't have the nice JavaScript thing where undefined = missing, so the below is actually broken when the third argument is not passed.

Reply:
Yeah, true that. Will fix.